plasmapy/atomic/atomic.py

```diff
@@ -1257,3 +1257,438 @@ def stable_isotopes_for_element(argument: Union[str, int],
+        string, case insensitive), or element name (e.g. "Francium", string,
+        case insensitive).
+
+    Returns
+    -------
+
+    block: string
+        The periodic table block of the element.
+
+    Raises
+    ------
+
+    TypeError:
+        If the argument is not a string or integer.
+
+    ValueError:
+        If the argument cannot be used to identify the element.
+
+    See also
+    --------
+
+        periodic_table_period: returns periodic table period of element.
+        periodic_table_group: returns periodic table group of element.
+
+    Examples
+    --------
+
+    >>> periodic_table_block(66)
+    'f'
+    >>> periodic_table_block("72")
+    'd'
+    >>> periodic_table_block("tl")
+    'p'
+    >>> periodic_table_block("thallium")
+    'p'
+    >>> periodic_table_block("FR")
+    's'
+    >>> periodic_table_block("FRANCIUM")
+    's'
+
+    """
+    if not isinstance(argument, (str, int)):
+        raise TypeError("The argument to periodic_table_block must be " +
+                        "either a string representing the element or its " +
+                        "symbol, or an integer representing its atomic " +
+                        "number.")
+    atomic_symbol = _get_atomic_symbol(argument)
+    block = _Elements[atomic_symbol]["block"]
+    return block
+
+
+def periodic_table_category(argument: Union[str, int]) -> str:
+    r"""Returns the periodic table category.
+
+    Parameters
+    ----------
+
+    argument: string or integer
+        Atomic number (either integer or string), atomic symbol (e.g. "H",
+        string, case insensitive), or element name (e.g. "Francium", string,
+        case insensitive).
+
+    Returns
+    -------
+
+    category: string
+        The periodic table category of the element.
+
+    Raises
+    ------
+
+    TypeError:
+        If the argument is not a string or integer.
+    ValueError:
+        If the argument cannot be used to identify the element.
+
+    See also
+    --------
+
+        periodic_table_period: returns periodic table period of element.
+        periodic_table_group: returns periodic table group of element.
+
+    Examples
+    --------
+
+    >>> periodic_table_category(82)
+    'Post-transition metals'
+    >>> periodic_table_category("85")
+    'Halogens'
+    >>> periodic_table_category("Ra")
+    'Alkaline earth metals'
+    >>> periodic_table_category("AC")
+    'Actinides'
+    >>> periodic_table_category("rhodium")
+    'Transition metals'
+
+    """
+    if not isinstance(argument, (str, int)):
+        raise TypeError("The argument to periodic_table_category must be " +
+                        "either a string representing the element or its " +
+                        "symbol, or an integer representing its atomic " +
+                        "number.")
+    atomic_symbol = _get_atomic_symbol(argument)
+    category = _Elements[atomic_symbol]["category"]
+    return category
+
+
+def _get_atomic_symbol(argument: Union[str, int]) -> str:
```

Comment:
It looks like there's some duplication here with `atomic_symbol()`, which is in `atomic/names.py` and has most of the same functionality as `_symbol_from_symbol()`, `_symbol_from_name()`, and `_symbol_from_number()`.  One difference is that the atomic symbols are case-sensitive in `atomic_symbol`, which helps reduce ambiguity from things like `'p'` being a proton and `'P'` being phosphorus.  We could probably use `atomic_symbol()` in the `periodic_table_*` functions to be consistent with the other atomic routines.